doc/i2p.md

```diff
@@ -47,13 +47,21 @@ information in the debug log about your I2P configuration and connections. Run
 `dash-cli help logging` for more information.
 
 It is possible to restrict outgoing connections in the usual way with
-`onlynet=i2p`. I2P support was added to Dash Core in version 20.0 (fall 2023)
+`onlynet=i2p`. I2P support was added to Dash Core in version 20.0 (fall-2023)
 and there may be fewer I2P peers than Tor or IP ones. Therefore, using
 `onlynet=i2p` alone (without other `onlynet=`) may make a node more susceptible
 to [Sybil attacks](https://en.dash.it/wiki/Weaknesses#Sybil_attack). Use
 `dash-cli -addrinfo` to see the number of I2P addresses known to your node.
 
-## I2P related information in Dash Core
```

Comment:
22589: I would consider this complete; we don't have a need for the other commit.